scripts/Muon/fft_presenter.py

```diff
@@ -100,6 +101,7 @@ def handleButton(self):
         self.thread.start()
         self.view.setPhaseBox()
 
+    # kills the thread at end of execution
     def handleFinished(self):
         self.activate()
         self.thread.deleteLater
```

Comment:
Is this really how the thread gets cleaned up? It is not actually a call?

Reply:
deleteLater is a slot, see https://stackoverflow.com/questions/28669195/qt-how-to-clean-up-a-qthread-when-you-do-not-subclass-it

Reply:
I know it's a slot, but I was merely wondering if you would actually need to call the attribute in Python, i.e. `self.thread.deleteLater()`. This is what I found people doing on the few examples I found on the Internet. Anyway, I'm not familiar with PyQt so I trust you know what you're doing here.